app/src/main/res/layout/fragment_notifications.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is this used anywhere?  Is there a Fragment class with event handling logic for this page?  
Only complete features, which meet the team's Definition of Done, should be merged to master.  If this feature is not complete, I would not merge this pull request.